src/hyperiond/hyperiond.cpp

```diff
@@ -170,29 +170,29 @@ void HyperionDaemon::freeObjects()
 	delete _jsonServer;
 	_flatBufferServer->thread()->quit();
 	_flatBufferServer->thread()->wait(1000);
-	_flatBufferServer->deleteLater();
-	_flatBufferServer->thread()->deleteLater();
+	delete _flatBufferServer->thread();
```

Comment:
Event loop is already finished at that point, hence the scheduled events will not work.